web/src/services/i18n/translations/ja.yml

```diff
@@ -72,6 +72,11 @@ Datasets: データセット
 Plugins: プラグイン
 Manage projects: プロジェクト管理
 Documentation: ヘルプ
+Choose Project Type: Choose Project Type
+classic: classic
+create project with classic UI: create project with classic UI
+Experimental: Experimental
```

Comment:
Beta please. We decided `beta` instead of `experimental`

Reply:
I kept it as design , but since you agreed on this I will change it with all later comments